tests/test_asttokens.py

```diff
@@ -146,6 +145,7 @@ def test_unicode_offsets(self):
 
   def test_coding_declaration(self):
     """ASTTokens should be able to parse a string with a coding declaration."""
+    # TODO: figure out if the comment below is relevant
```

Comment:
The Python 2 part can be removed, but let's test that it works with both `bytes` and `str`.